src/subscription/subscribe.js

```diff
@@ -171,19 +172,28 @@ function subscribeImpl(
             reportGraphQLError,
           )
         : ((resultOrStream: any): ExecutionResult),
-    reportGraphQLError,
```

Comment:
@IvanGoncharov The removal of this "catch" handler means that the existing tests for error handling in `subscribe` already do exercise the new logic. Do you still think explicit new tests are needed?

Reply:
@taion I mean you change a code to change how certain errors handled and no test cases were broken. It means that there no test cases covering this behaviour and without test cases, we can't be sure it's working as expected.

Plus coverage for `subscribe.js` decreased from 100% in this PR:
https://coveralls.io/builds/18460657/source?filename=src/subscription/subscribe.js#L121

Reply:
Ah, I didn't see the Coveralls check. I'll add explicit test coverage then.